docs/anet-dictionary.yml.production.template

```diff
@@ -0,0 +1,250 @@
+dictionary:
```

Comment:
the difference between `anet-dictionary.yml.production.template` and `anet-dictionary.yml.template` was mostly intended to be in the non-dictionary part. I suggest to not add a `docs/anet-dictionary.yml.production.template` as it is not needed - it should eventually be identical to `docs/anet-dictionary.yml.template`

Reply:
docs/anet-dictionary.yml.production.template is removed